services/QuillLMS/client/app/bundles/Staff/components/evidence/activitySessions/promptTable.tsx

```diff
@@ -37,12 +39,15 @@ const PromptTable = ({ activity, rules, prompt, showHeader, sessionId }: PromptT
         });
       });
       completed = usedAttempts.some((attempt) => attempt.optimal);
+      optimal = completed ? true : false
```

Comment:
Doesn't `.some` return a boolean? If it does this line should be unnecessary (you're just setting optimal to the value of completed)

Reply:
oh good catch, I'll update